client/experimentalclient/backup.go

```diff
@@ -0,0 +1,65 @@
+package experimentalclient
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path"
+
+	"github.com/coreos/etcd-operator/pkg/backup"
+	"github.com/coreos/etcd-operator/pkg/util/k8sutil"
+)
+
+type Backup interface {
+	// Request requests a backup. Request returns nil, once a backup is made successfully.
+	// Or it returns an error.
+	Request(ctx context.Context) error
+}
+
+type backupClient struct {
+	client *http.Client
+	url    string
+}
+
+func NewBackup(c *http.Client, clusterName string) Backup {
```

Comment:
It's fine as long as you want to have k8sutil in this package. Otherwise, we should remove this func.